sky/skylet/providers/scp/config.py

```diff
@@ -50,7 +50,7 @@ def bootstrap_instance_config(self, node_config):
         instance_config['serverType'] = node_config['InstanceType']
         instance_config['contractId'] = "None"
         instance_config['initialScript'] = self._get_vm_init_script(
-            node_config['auth']['ssh_public_key'])
+            node_config['AuthorizedKey'])
```

Comment:
Did we pick AuthorizedKey on our own, or is it a convention?

Reply:
I guess my question is it seems a bit weird that the different *.j2 use different field names for this.

One option is to conform to AWS's j2, i.e., cloud-init's "ssh_authorized_keys" field?

Reply:
I think the problem is that we want to keep the naming style the same as the other keys in the yaml file. For example, in `oci-ray.yml.j2`, all the keys are camel-case. Having a snake-case `ssh_authorized_keys` among them might be a bit weird?